Assets/Scripts/YarnSpinner/Code/YarnFiles/Phase7CallToAdventure.yarn.txt

```diff
@@ -0,0 +1,29 @@
+title: Phase7_CallToAdventure
+tags: 
+colorID: 0
+position: 497,104
+---
+<<setsprite FaceImage malphas>>
+    Malphas: I can get you the answers you’re looking for, even if you don’t realise it yet. Or will you go on with your pathetic life and eventually drink yourself to death? 
+    <<startPressureTimer DialoguePhase7_CallToAdventure 4,2>>
+        [[Yeah, whatever.|Option1]]
+        [[Maybe I don't want answers? |Option2]]
+===
+
+title: Option1
+tags: 
+colorID: 0
+position: 315,367
+---
+<<setsprite FaceImage devon>>
+    Devon: Yeah, whatever.
+===
+
+title: Option2
+tags: 
+colorID: 0
+position: 682,367
+---
+<<setsprite FaceImage devon>>
+    Devon: Maybe I don't want answers? 
```

Comment:
should be `Devon: Maybe I don't want answers? I’ve tried looking for answers. I’ve done everything I can. And nothing will bring her back.`

Reply:
After this as well there is a section of dialogue depending on the players decision. That section should display.

```
After acceptance:

Malphas (in Devon’s thoughts): By the way, my name is Malphas. A pleasure to make my introductions. 

Devon (thought): Great it even has a name.

After Refusal 

Malphas: If that is what you want. But remember, my offer still stands when you change your mind eventually.
```